Gems/LmbrCentral/Code/Tests/SpawnerComponentTest.cpp

```diff
@@ -106,13 +106,13 @@ class SpawnerApplication
 };
 
 class SpawnerComponentTest
-    : public testing::Test
+    : public UnitTest::AllocatorsTestFixture
```

Comment:
If this was changed to inherit from UnitTest::ScopedAllocatorSetupFixture, then the SystemAllocator Create call can be removed.